lib/cacheWrapper.js

```diff
@@ -1,76 +1,116 @@
 'use strict';
 
 var cacheWrapper = module.exports = {};
-
-var _ = require( 'lodash' );
-var Catbox = require( 'catbox' );
-var catboxMemory = require( 'catbox-memory' );
-var catboxRedis = require( 'catbox-redis' );
-var Q = require( 'q' );
-
+var _ = require('lodash');
+var Catbox = require('catbox');
+var catboxRedis = require('catbox-redis');
+var redisWrapper = require('./redisWrapper');
+var Q = require('q');
 var cacheClient;
 var startingClient = false;
 var queryQueue = [];
 var policies = [];
+var partition = '';
+
+cacheWrapper.initialise = function(serverConfig, cachePolicies) {
+  // no config, no cache...
+  if (_.isEmpty(serverConfig) || _.isEmpty(serverConfig.partition)) {
+    return Q.reject('server config missing partition');
```

Comment:
I like how you've referenced the variable in the `cachePolicies` rejection reason and think it would be nice to keep the consistency here;
```
return Q.reject('serverConfig or serverConfig.partition missing');
```